app/views/spree/user_registrations/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<% @body_id = 'signup' %>
```

Comment:
do we need to keep these `@body_id` for any reason?

Reply:
Probably not, I'll see what happens if we remove it.